src/application/PasswordGenerator.h

```diff
@@ -0,0 +1,31 @@
+#ifndef SRC_PASSWORDGENERATOR_H
+#define SRC_PASSWORDGENERATOR_H
+
+#include "PseudoRandomNumberGenerator.h"
+#include "PlaintextPassword.h"
+#include <iostream>
+
+namespace DDD::Services {
+    class PasswordGenerator {
+    public:
+        PasswordGenerator(PseudoRandomNumberGenerator *prng) : prng(prng) {}
+
+        DDD::ValueObjects::PlaintextPassword getRandomPassword() {
```

Comment:
imo, this should accept a length parameter, since users may prefer certain password lenghts

Reply:
Yep I thought so too. But then I thought that we probably shouldn't just have a parameter of type `int` but instead of type `PasswordLength` to comply with Domain Driven Design principles. Then I thought that we would also need to ask the user what password length he wants to actually make use of the parameter. And then I thought to myself that this would be too much work :D
However after you also suggested that I guess it's worth it.